test/test-utils/src/ErrorContract.s.sol

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
```

Comment:
consider adding this contract to src/contracts/* too as that's where they will eventually all be